Remotion/ObjectBinding/Web/UI/Controls/BocTreeView.cs

```diff
@@ -384,7 +396,9 @@ private void EvaluateTreeNode (WebTreeNode node)
 
     private void CreateAndAppendBusinessObjectNodeChildren (BusinessObjectTreeNode businessObjectNode)
     {
-      IBusinessObjectWithIdentity businessObject = businessObjectNode.BusinessObject;
+      IBusinessObjectWithIdentity? businessObject = businessObjectNode.BusinessObject;
+
+      Assertion.IsNotNull (businessObject, "The business object of the node must not be null.");
```

Comment:
How would I word the message then?

Reply:
Wtih a lot of null-fallbacks, I guess. 

=> So, yeah, we leave it. This was more of a commentary on the state of things, sorry. And in case you've got a good idea because I don't.